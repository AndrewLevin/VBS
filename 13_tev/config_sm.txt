mode=sm

#background_file=filename,process name,qcd uncertainty mode, pdf uncertainty mode
background_file=/data/blue/anlevin/ntuples/wpwpjj_qcd_v10.root,wpwpjjqcd,syscalc
background_file=/data/blue/anlevin/ntuples/ttbar_v10.root,ttbar,none

signal_file=/data/blue/anlevin/ntuples/wpwpjj_ewk_v10.root

lumi=10

variable=mjj

outfile=histograms.root

datacard_base=datacard

channel=all

charge=both
