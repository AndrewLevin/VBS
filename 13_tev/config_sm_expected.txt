mode=sm_mc_fake

#background_file=/data/blue/anlevin/ntuples/wpwpjj_qcd_v101.root,wpwpjjqcd,none
background_file=/afs/cern.ch/work/a/anlevin/data/flattish_ntuples/wzjj_highmll_v1.root,wzjj,none
#background_file=/data/blue/anlevin/ntuples/wjwj-dps_v101.root,wjwjdps,none
#background_file=/data/blue/anlevin/ntuples/zjets_v201_skim.root,zjets,none
background_file=/afs/cern.ch/work/a/anlevin/data/flattish_ntuples/wgjj_v1.root,wgjets,none

signal_file=/afs/cern.ch/work/a/anlevin/data/flattish_ntuples/wpwpjj_ewk_qcd_v1.root

data_file=/afs/cern.ch/work/a/anlevin/data/flattish_ntuples/run2015_v1_skim.root

lumi=2.215

variable=mjj

outfile=histograms.root

datacard_base=datacard

channel=all

charge=both
